Nil pointer dereference on vpn client

**Describe the bug**
There is a panic on starting vpn-client if vpn-client is started before the stun module is initilised. 

**Environment information:**

- OS: e.g. Linux
- Platform: the equivalent of `uname -smr` or `uname -smrv` e.g. Darwin 17.4.0 x86_64

**Steps to Reproduce**
Steps to reproduce the behavior:

1. Run `./skywire-visor -c skywire-config.json`
2. Connect to a VPN server 
4. See error

**Actual behavior**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xb1db93]

goroutine 274 [running]:
github.com/skycoin/skywire/pkg/visor.getRouteSetupHooks.func1({0x3, 0xe6, 0xa4, 0x35, 0xb1, 0x6d, 0xe6, 0xac, 0x49, 0xd7, ...}, ...)
        /home/erson/Workspace/Go/skywire/pkg/visor/init.go:517 +0x413
github.com/skycoin/skywire/pkg/router.(*router).DialRoutes(0xc0005500b0, {0x1381bb0, 0xc00003a0b0}, {0x3, 0xe6, 0xa4, 0x35, 0xb1, 0x6d, 0xe6, ...}, ...)
        /home/erson/Workspace/Go/skywire/pkg/router/router.go:253 +0xd54
github.com/skycoin/skywire/pkg/app/appnet.(*SkywireNetworker).DialContext(0xc0005013e0, {0x1381bb0, 0xc00003a0b0}, {{0xc0028769f8, 0x6}, {0x3, 0xe6, 0xa4, 0x35, 0xb1, ...}, ...})
        /home/erson/Workspace/Go/skywire/pkg/app/appnet/skywire_networker.go:60 +0x16b
github.com/skycoin/skywire/pkg/app/appnet.DialContext({0x1381bb0, 0xc00003a0b0}, {{0xc0028769f8, 0x6}, {0x3, 0xe6, 0xa4, 0x35, 0xb1, 0x6d, ...}, ...})
        /home/erson/Workspace/Go/skywire/pkg/app/appnet/networker.go:82 +0x96
github.com/skycoin/skywire/pkg/app/appnet.Dial(...)
        /home/erson/Workspace/Go/skywire/pkg/app/appnet/networker.go:72
github.com/skycoin/skywire/pkg/app/appserver.(*RPCIngressGateway).Dial(0xc0025d4c60, 0xc0028d6000, 0xc002876a00)
        /home/erson/Workspace/Go/skywire/pkg/app/appserver/rpc_ingress_gateway.go:117 +0x208
reflect.Value.call({0xc000422180, 0xc00054afe8, 0x13}, {0xca471a, 0x4}, {0xc0027b3ef8, 0x3, 0x3})
        /usr/local/go/src/reflect/value.go:543 +0x814
reflect.Value.Call({0xc000422180, 0xc00054afe8, 0x0}, {0xc00273b6f8, 0x3, 0x3})
        /usr/local/go/src/reflect/value.go:339 +0xc5
net/rpc.(*service).call(0xc0025f6080, 0x0, 0x0, 0xc0025e42f0, 0xc00029fe00, 0xc00273b7d0, {0xbe3d40, 0xc0028d6000, 0xc00054ae70}, {0xb7b3e0, ...}, ...)
        /usr/local/go/src/net/rpc/server.go:377 +0x239
created by net/rpc.(*Server).ServeCodec
        /usr/local/go/src/net/rpc/server.go:474 +0x405
```

--- a/pkg/visor/init.go
+++ b/pkg/visor/init.go
@@ -514,7 +514,8 @@ func getRouteSetupHooks(ctx context.Context, v *Visor, log *logging.Logger) []ro
 			for _, trans := range transports {
 				ntype := network.Type(trans)
 				// skip if SUDPH is under symmetric NAT / under UDP firewall.
-				if ntype == network.SUDPH && (v.stunClient.NATType == stun.NATSymmetric || v.stunClient.NATType == stun.NATSymmetricUDPFirewall) {
+				if ntype == network.SUDPH && v.stunClient != nil && (v.stunClient.NATType == stun.NATSymmetric ||
+					v.stunClient.NATType == stun.NATSymmetricUDPFirewall) {
 					continue
 				}
 				err := retrier.Do(ctx, func() error {

